appveyor.yml

```diff
@@ -9,21 +9,44 @@ environment:
     - PYTHON_VERSION: 2.7
       PYTHON: C:\Miniconda-x64
       ENV: miniconda
+      TEST_TOX_ENV: py27
+      COVERAGE_TOX_ENV: coverage-py27
+      BUILD_TOX_ENV: build-py27
```

Comment:
How do you ensure that tox will run into the miniconda env ? 

References:
* https://stackoverflow.com/questions/30555943/is-it-possible-to-use-tox-with-conda-based-python-installations
* https://github.com/tox-dev/tox/tree/master/doc/drafts and also `https://github.com/tox-dev/tox/issues/273`